config/runtimes/vllm-gaudi-template.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: vLLM ServingRuntime to support Gaudi(for Habana AI processors)
+    openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/vllm
+    template.openshift.io/long-description: This template defines resources needed to deploy vLLM ServingRuntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: 'REST'
+  name: vllm-gaudi-runtime-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: vllm-gaudi-runtime
+      annotations:
+        openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
+        opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+      labels:
+        opendatahub.io/dashboard: 'true'
+    spec:
+      annotations:
+        prometheus.io/port: '8080'
+        prometheus.io/path: '/metrics'
+      multiModel: false
+      supportedModelFormats:
+        - autoSelect: true
+          name: vLLM
+      builtInAdapter:
+        modelLoadingTimeoutMillis: 90000
+      containers:
+        - name: kserve-container
```

Comment:
does it need any probe? like readiness/liveness?

Reply:
We do not set probes because of historical issues:
1. Probes were problematic in  kserve until recently
2. We cannot set arbitrary probe values: they might work for all deployments since deployment time varies depending on model size and hardware.

We'll probably get back to this in the near future

Reply:
This is Jira where we decided to remove probes
https://issues.redhat.com/browse/RHOAIENG-8736

Reply:
ok this is runtime team decision so I have no objection.